pretix_extended_api/views/tickets.py

```diff
@@ -46,3 +51,43 @@ def attendee_has_ticket(self, request, **kwargs):
 
         qs = qs.filter(events_filter)
         return Response({"user_has_admission_ticket": qs.exists()})
+
+    @scopes_disabled()
+    @action(url_path="attendee-tickets", detail=False, methods=["get"])
+    def attendee_tickets(self, request, *args, **kwargs):
+        check_permission(request, "can_view_orders")
+
+        serializer = AttendeeTicketBodySerializer(data=request.query_params)
+        serializer.is_valid(True)
+
+        attendee_email = serializer.data["attendee_email"]
+
+        qs = OrderPosition.objects.filter(
```

Comment:
we should filter for event here, we do not want all the tickets of the user across all the events

Reply:
you should have the organiser/event in the URL

Reply:
I added a filter by events